txbits/app/wallet/WalletModel.scala

```diff
@@ -154,10 +162,21 @@ class WalletModel(val db: String = "default") {
     SQL""" select create_cold_storage_transfer($txId, $address, ${value.bigDecimal}) """.execute()
   }
 
+  def getColdStorageTransfer(txId: Long) = DB.withConnection(db) { implicit c =>
+    SQL""" select * from get_cold_storage_transfer($txId) """().map(row => row[Option[String]]("address") -> row[Option[BigDecimal]]("value")).head match {
+      case (Some(address: String), Some(value: BigDecimal)) => Some(address, value)
+      case _ => None
+    }
+  }
+
   def setWithdrawalTxHashMutated(txId: Long, txHash: String) = DB.withConnection(db) { implicit c =>
     SQL""" select set_withdrawal_tx_hash_mutated($txId, $txHash) """.execute()
   }
 
+  def getWithdrawConstantFee(currency: CryptoCurrency) = DB.withConnection(db) { implicit c =>
+    SQL""" select get_withdraw_constant_fee(${currency.toString}) """().map(row => row[BigDecimal]("get_withdraw_constant_fee")).head
```

Comment:
This number is arbitrary and we should not be using it. We should use whatever is set as `maxtxfee` in bitcoin.conf.